haml_lint.gemspec

```diff
@@ -21,9 +21,10 @@ Gem::Specification.new do |s|
 
   s.required_ruby_version = '>= 2.2.0'
 
+  s.add_development_dependency 'rake', '>= 10', '< 13'
+  s.add_development_dependency 'rubocop', '>= 0.50.0'
```

Comment:
RuboCop is not a development dependency&mdash;HAML-Lint requires it for the [RuboCop](https://github.com/brigade/haml-lint/blob/master/lib/haml_lint/linter/rubocop.rb) linter.

The development dependency is defined in the [Gemfile](https://github.com/brigade/haml-lint/blob/a2e85796e23b25623d9cc5cffee2074ef5fc0760/Gemfile#L15).

Reply:
I actually don't know how I forgot that, I'm so used to rubocop being used internally for linting only and went on auto pilot. Will revert that change. Apologies.

Reply:
Are you happy to move rake into the Gemfile then?

Reply:
Apologies for this long PR process, have removed that commit and amended the PR body.